api/groa_ds_api/utils.py

```diff
@@ -73,15 +73,16 @@ def _get_JSON(self, rec_df):
             rec_json = []
 
             for i in range(rec_df.shape[0]):
-                curr_object = dict(rec_df.iloc[i].to_dict())
+                rec = dict(rec_df.iloc[i].to_dict())
+                rec['score'] = float(rec['score']) if not isinstance(rec['score'], str) else 0.0
```

Comment:
If `score` is not a str what type are you expecting it to be?
If `score` is a str it's always going to be set to 0? Can you elaborate on what this is doing?

Reply:
This is just because the validator earlier sets them to a string "None" and FastAPI errors on a str so Ben is (from what I understand) changing them to a float so that it will not error out.